eclair-core/src/main/scala/fr/acinq/eclair/PluginParams.scala

```diff
@@ -34,6 +35,9 @@ trait CustomFeaturePlugin extends PluginParams {
   /** Feature bit that the plugin wants to advertise through Init message. */
   def feature: Feature
 
+  /** Once disconnect happens, should Eclair attempt periodic reconnects to a given remote node even if there is no normal channels left */
+  def forceReconnect(nodeId: PublicKey): Boolean
```

Comment:
That doesn't make sense in the `CustomFeaturePlugin`, it's not feature-related.
You should create a new trait for that.

Reply:
OK, my rationale was that `CustomFeaturePlugin` defines network messages it's interested it so it's inherently something network-related, might as well care about connection.